docs/pages/cloud/getting-started.mdx

```diff
@@ -29,23 +33,58 @@ Login into Teleport and test the connection:
 ```bash
 # tsh logs you in and receives short-lived certificates
 $ tsh login --proxy=myinstance.teleport.sh:443 --user=email@example.com
-# tctl is a cluster configuration tool
+# Your initial role is admin as the creater of the account
+# tctl is a cluster configuration tool. Confirm the status of the cluster
 $ tctl status
 ```
 
-## Step 3/3 Explore your cluster
+## Step 3/5 Explore your cluster
 
 List nodes in the cluster:
 
 ```bash
 $ tctl nodes ls
 ```
+Next we'll add a node so you can access it remotely.
 
+## Step 4/5 Install the Teleport agent on a Linux Server
 Generate a [dynamic token](../admin-guide.mdx#short-lived-dynamic-tokens):
 
 ```bash
-$ tctl nodes add --ttl=5m --roles=node --token=$(uuid)
+$ tctl nodes add --ttl=20m --roles=node --token=$(uuid)
+```
+
+Install Teleport on a Linux server
```

Comment:
What's the benefit of this compared to our wizard?

Reply:
@klizhentas  I can change to using the wizard.  Will take out the nodes add part then.  Will add screen shots then.